app/javascript/packs/components/FilterBar.jsx

```diff
@@ -0,0 +1,153 @@
+import React, { Component } from 'react';
+import SelectField from 'material-ui/SelectField'
+import Slider from 'material-ui/Slider';
+import MenuItem from 'material-ui/MenuItem';
+import Checkbox from 'material-ui/Checkbox';
+import PropTypes from 'prop-types';
+
+const styles = {
+  customFont: {
+    color: 'white',
+  },
+  customSlider : {
+    width: 250,
+    marginBottom: 0,
+    marginTop: 0,
+  },
+  customCheckbox: {
+    marginTop: 12,
+  },
+};
+
+class MinimalRating extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      ratingStars: [false, false, false],
+      rating: 0,
+    };
+
+    this.handleCheckboxChange = this.handleCheckboxChange.bind(this);
+  }
+
+  handleCheckboxChange(type, index) {
+    if (type === 'rating') {
+      let stars = [false, false, false];
+      let rating = 0;
+      // when only one star is true - clear all stars
+      if (!((index === 1) && this.state.ratingStars[0] && !this.state.ratingStars[1])) {
+        for (let i = 0; i < index; i = i + 1) {
+          stars[i] = true;
+          rating = index;
+        }
+      }
+      this.setState({ ratingStars: stars, rating: rating });
+      this.props.updateRating(rating);
+    }
+  }
+
+  render() {
+    const ratingStars = this.state.ratingStars;
+    return(
+      <div className="minimal-rating">
+        <span>{'Select Minimal Rating'}</span>
+        <div className="stars">
+          <Checkbox
```

Comment:
these 3 can be created using map:
{ratingStars.map((rating) => {
    return(
        <Checkbox
        checked={rating} 
...

Reply:
👍